### DISCUSSION TITLE
Bungie provider is not authenticating

### DISCUSSION THREAD

**Discussion by smashboy** [nextauthjs/next-auth#1396] [Help]
Hi everyone! I'm trying to implement auth through bungine.net and instead of session creation I'm only getting auth code & state: 
![image](https://user-images.githubusercontent.com/30213662/109427265-39984f00-79fa-11eb-8677-c88d1edca8bb.png)

I'm probably messing smth up in setup  and don't know what exactly.

**Comment by balazsorban44** [comment]
a reproduction repo could help us help you. the information you are providing is nothing to go on, unfortunately. 🙁

**Reply by smashboy** [reply]
Here is a repo: https://github.com/smashboy/next-auth-bungie
I was following this instructions: https://next-auth.js.org/providers/bungie

There is index page with auth button and api/[...nextauth] with custom provider

**Reply by smashboy** [reply]
@balazsorban44 any updates?
